docs/Rapid/4-Keyper Manual/4-Adaptive Designer/6-How to configure connection of root section with data table/6-How to configure connection of root section with data table.md

```diff
@@ -1 +1,31 @@
-# How to configure connection of root section with data table?
\ No newline at end of file
+# How to configure connection of root section with data table?
+
+Establishing a connection of a section with a data table enables the system to fetch and / or feed data to the data table from the form.   
+  
+Please consider the below steps for establishing connection of root section with data table:  
+
+1. Select the  <a href="https://rapiddocs.z8.web.core.windows.net/docs/Rapid/Keyper%20Manual/Adaptive%20Designer/Understanding%20the%20Overview%20Tab/" target="_blank">**Overview tab**</a> for the desired Adaptive Document. 
```

Comment:
We've moved the docs to the proper domain, so we can use `href="https://docs.rapidplatform.com/docs/{etc}"` for href linknow